projects/github.com/fukuchi/package.yml

```diff
@@ -24,7 +24,8 @@ build:
   script: |
     ./autogen.sh
     ./configure --disable-dependency-tracking --prefix={{prefix}} $ARGS
-    VERSION={{version}} make
+    '#define VERSION "{{version}}"' >> config.h
+    make
     make --jobs {{ hw.concurrency }} install
```

Comment:
Usually, this can just be `make --jobs {{ hw.concurrency }} install`, since most people make `install` depend on the build; if they need to be separate, the jobs multiplier should be on the build (it's much less interesting for `install`).